internal/delivery/http/story_handler.go

```diff
@@ -125,3 +158,51 @@ func (s *StoryHandler) CreateStory(c echo.Context) error {
 		Message: "Success Create Story",
 	})
 }
+
+func (s *StoryHandler) UpdateStory(c echo.Context) error {
+	storyIdParam := c.Param("id")
+
+	storyId, err := strconv.ParseInt(storyIdParam, 10, 64)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, response{
+			Status:  http.StatusBadRequest,
+			Message: "Invalid story ID",
+		})
+	}
+
+	var input model.UpdateStoryInput
+	if err := c.Bind(&input); err != nil {
+		return c.JSON(http.StatusBadRequest, response{
+			Status:  http.StatusBadRequest,
+			Message: err.Error(),
+		})
+	}
+
+	if err := s.storyUsecase.Update(c.Request().Context(), storyId, input); err != nil {
+		return c.JSON(http.StatusInternalServerError, response{
+			Status:  http.StatusInternalServerError,
+			Message: err.Error(),
+		})
+	}
+
+	return c.JSON(http.StatusOK, response{
+		Status:  http.StatusOK,
+		Message: "Success Update Story",
+	})
+}
+
+func (s *StoryHandler) DeleteStory(c echo.Context) error {
+	id := c.Param("id")
+	parsedId, err := strconv.Atoi(id)
+	if err != nil {
+		return echo.NewHTTPError(500, err.Error())
+	}
+
+	if err := s.storyUsecase.Delete(c.Request().Context(), int64(parsedId)); err != nil {
+		return echo.NewHTTPError(500, err.Error())
+	}
+
+	return c.JSON(http.StatusNoContent, response{
+		Status: "No Content",
+	})
```

Comment:
```suggestion
	return c.JSON(http.StatusNoContent)
```
No content means blank (there is not content in http response)